virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc

```diff
@@ -15,6 +15,9 @@ include::modules/virt-understanding-smart-cloning.adoc[leveloffset=+1]
 
 include::modules/virt-cloning-a-datavolume.adoc[leveloffset=+1]
 
+[id="{context}_additional-resources"]
 == Additional resources
 
 * xref:../../../virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume.adoc#virt-cloning-pvc-of-vm-disk-into-new-datavolume_virt-cloning-vm-disk-into-new-datavolume[Cloning the persistent volume claim of a virtual machine disk into a new data volume]
+
+* xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Configure preallocation mode] to improve write performance for data volume operations.
```

Comment:
In https://issues.redhat.com/browse/CNV-6680, in the description it states that preallocation is not valid for cloning:

From the description:
-----------------------------------------------
As a CNV user I want to enable preallocation of my kubevirt disk via an API in the DataVolume spec.

`preallocation: true|false`

    Valid only for kubevirt content-type
    Valid for import (http and registry) and Upload
    Valid for block mode and filesystem mode
    **Not valid for clone**